internal/http/services/owncloud/ocdav/dav.go

```diff
@@ -39,6 +38,10 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+const (
+	_trashbinPath = "trash-bin"
```

Comment:
Curious: could you explain (or point me to where is explained) when to use underscores as constant prefixes and when not?

Reply:
I like to prefix unexported package constants with an underscore. That way you immediately see that it's not a local variable. And you don't accidentally shadow those variables.

Reply:
I like the uber style guide for example: https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_

Reply:
Cool. Thanky!